provisioning/playbook.yml

```diff
@@ -40,7 +40,7 @@
     - { role: geerlingguy.apache, when: drupalvm_webserver == 'apache', tags: ['webserver']}
     - { role: geerlingguy.apache-php-fpm, when: drupalvm_webserver == 'apache', tags: ['webserver'] }
     - { role: geerlingguy.nginx, when: drupalvm_webserver == 'nginx', tags: ['webserver'] }
-    - { role: geerlingguy.php-versions, tags: ['php', 'xdebug', 'database'] }
+    - { role: geerlingguy.php-versions, when: php_version != '', tags: ['php', 'xdebug', 'database'] }
```

Comment:
Egregious haha, word of the day. Hmm maybe we should add `php_version|default('')` in the few places we use it? I think it's the dashboard and php extension workspaces that uses it. Then it would work with the value being `false`, `no` and `''`.

Doing `when: php_version` and then later on `workspace: "/root/php{{ php_version }}"` definitely raises an eyebrow for me